linkis-public-enhancements/linkis-basedata-manager/src/main/java/org/apache/linkis/basedatamanager/server/utils/UdfTreeUtils.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.basedatamanager.server.utils;
+
+import org.apache.linkis.basedatamanager.server.domain.UdfTreeEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UdfTreeUtils {
+
+  /** BuildTree 构建树形结构 */
+  public List<UdfTreeEntity> udfTreeList = new ArrayList<>();
+
```

Comment:
Use English annotations or
Chinese and English annotations

Reply:
done